src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/DotNetBuildCommandHelper.cs

```diff
@@ -37,10 +39,12 @@ internal class DotNetBuildCommandHelper
             var stdOutMsgs = new List<string>();
             var stdErrorMsgs = new List<string>();
 
-            //TODO Remove this when CLI merges build3 to build.
-            var buildCommandName = Path.GetExtension(project).Equals(".csproj")
-                ? "build3"
-                : "build";
+            // HACK :: Till we upgrade the repo to use msbuild based CLI, use *3 verbs for the tests.
+            if (System.Environment.GetEnvironmentVariable("USE_3_VERBS").Equals("1")
```

Comment:
This'll explode when you publish and use outside of our CI 😄 .

`string.Equals(Environment.GetEnvironmentVariable("USE_3_VERBS"), "1", StringComparer.Ordinal)`.

I prefer to always use `string.Equals` instead of `.Equals` because it works well with `null`.